layouts/theme.liquid

```diff
@@ -82,9 +96,27 @@
     {% include 'accessibility-menu' %}
 
     {% if current_path != '/express_checkout/payment' and current_path != '/express_checkout/success' %}
-      {% include 'navbar' %}
+      {% comment %} Teste logico para saber qual pagina está sendo mostrada e se foi escolhido mostrar ou não o objeto {% endcomment %}
+      {% if settings.view.config_pages.home.header == '1' or settings.view.config_pages.check.header == '1' or settings.view.config_pages.login.header == '1' %}
```

Comment:
@evertonEADBOX pelo que vejo os attributos dessa condição está sendo retestados novamente ao acessar ser verdadeira, poderia ser usado o controle [elseif/else](https://shopify.github.io/liquid/tags/control-flow/) conforme documentação.
Pelo que eu entendi, se nao for nenhuma das condicões testadas a navbar é exibida. Se não tiver nada a ver comenta aqui :)

Reply:
Foi necessário realizar vários testes pois se usar apenas a condição elseif/else há casos nos quais o header fica duplicado pois duas ou mais condições acabam sendo verdade.
Fiz alguns testes e não consegui chegar em uma condição elseif que não ocorram o problema da duplicidade em pelo menos um caso.
Essa primeira linha é para verificar globalmente se há a presença de alguma config caso tenha nos testes abaixo que serão definidos quais telas mostrar ou não o objeto.